### PR TITLE
Support modbus single register writes and long reads

### LINKED ISSUE #525: SerialCommHub doesn't support write_single_register command
As far as I can see the re is no way to send command WRITE_SINGLE_REGISTER to the Modbus device.

Theoretically, it it could be done by "write multiple registers" with count 1. But behaviour of the remote device can be different depending on command that is used. 

To my knowledge some functionality of [Iskra meter](https://www.iskra.eu/de/Energy-meters_2/Energy-meter-WM3M4-and-WM3M4C_1/) can be only used with WRITE_SINGLE_REGISTER command.

### PR DESCRIPTION
**Pull request by golovasteek**
Resolves #525 
Supersedes #441

### CODE REVIEW

File: interfaces/serial_communication_hub.yaml [hunk 1]
```diff
@@ -72,6 +72,30 @@ cmds:
       description: Status code of the transfer
       type: string
       $ref: /serial_comm_hub_requests#/StatusCodeEnum
+  modbus_write_signle_register:
```
Comment (barsnick) [inline interfaces/serial_communication_hub.yaml:75]: signle -> single

**Comment by corneliusclaussen** [comment]
Supersedes https://github.com/EVerest/everest-core/pull/441: Should we close #441 then?

**Review by corneliusclaussen — APPROVED** [review]
Should be squashed before merging with proper commit description

File: interfaces/serial_communication_hub.yaml (hunk 1 above)
Comment (golovasteek) [reply interfaces/serial_communication_hub.yaml]: Thanks. FIxed.

**Pull request merged**